types/resource_selector.go

```diff
@@ -24,6 +25,8 @@ type ResourceSelector struct {
 	//  'max-age=X' (cache for X duration)
 	Cache string `yaml:"cache,omitempty" json:"cache,omitempty"`
 
+	IncludeDeleted bool `yaml:"includeDeleted" json:"includeDeleted"`
```

Comment:
This should probably be yaml ignored as it doesn't make sense for a user to specify it

Reply:
Done